lib/App/DuckPAN/Cmd/Test.pm

```diff
@@ -6,27 +6,44 @@ with qw( App::DuckPAN::Cmd );
 
 use MooX::Options protect_argv => 0;
 
+use Path::Tiny;
+
 option full => (
-	is          => 'ro',
-	lazy        => 1,
-	short       => 'f',
-	default     => sub { 0 },
-	doc         => 'run full test suite via dzil',
+    is      => 'ro',
+    lazy    => 1,
+    short   => 'f',
+    default => sub { 0 },
+    doc     => 'run full test suite via dzil',
 );
 
 sub run {
-	my ( $self ) = @_;
-
-	my $ret = 0;
-
-	if ($self->full) {
-		$self->app->emit_error("Could not find dist.ini.") unless -e "dist.ini";
-		$self->app->emit_error("Could not begin testing. Is Dist::Zilla installed?") if $ret = system("dzil test");
-	} else {
-		$self->app->emit_error("Tests failed! See output above for details") if $ret = system("prove -Ilib");
-	}
-
-	return $ret;
+    my ($self, @args) = @_;
+
+    my $ret = 0;
+
+    if ($self->full) {
+        $self->app->emit_error("Could not find dist.ini.") unless -e "dist.ini";
+        $self->app->emit_error("Could not begin testing. Is Dist::Zilla installed?") if $ret = system("dzil test");
+    } else {
```

Comment:
@zachthompson This the general preference? There seems to be quite a few files with cuddled elses. Want me to de-cuddle them all in #309?